src/amo/components/RatingManager/index.js

```diff
@@ -18,34 +21,52 @@ import {
 import translate from 'core/i18n/translate';
 import log from 'core/logger';
 import DefaultRating from 'ui/components/Rating';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { UserReviewType } from 'amo/actions/reviews';
+import type { SubmitReviewParams } from 'amo/api';
+import type { UrlFormatParams } from 'core/api';
+import type { ApiStateType } from 'core/reducers/api';
+import type { DispatchFn } from 'core/types/reduxTypes';
+import type { AddonType, AddonTypeProp, AddonVersionType }
+  from 'core/types/addonTypes';
 
 import './styles.scss';
 
+type LoadSavedReviewFn = ({|
+  userId: number,
+  addonId: number,
+|}) => Promise<*>;
+
+type SubmitReviewFn = (SubmitReviewParams) => Promise<void>;
+
+type RatingManagerProps = {|
+  AddonReview: typeof DefaultAddonReview,
+  AuthenticateButton: typeof DefaultAuthenticateButton,
+  Rating: typeof DefaultRating,
+  addon: AddonType,
+  errorHandler: ErrorHandlerType,
+  apiState: ApiStateType,
+  i18n: Object,
```

Comment:
Yeah, in a future patch I'd like to define `i18n` because we rely on `i18n.gettext(...)` everywhere.